operator/pkg/model/ingestion/ingress.go

```diff
@@ -90,35 +90,37 @@ func Ingress(ing slim_networkingv1.Ingress, defaultSecretNamespace, defaultSecre
 		}
 
 		l.Hostname = host
-		for _, path := range rule.HTTP.Paths {
-
-			route := model.HTTPRoute{}
-
-			switch *path.PathType {
-			case slim_networkingv1.PathTypeExact:
-				route.PathMatch.Exact = path.Path
-			case slim_networkingv1.PathTypePrefix:
-				route.PathMatch.Prefix = path.Path
-			case slim_networkingv1.PathTypeImplementationSpecific:
-				route.PathMatch.Regex = path.Path
-			}
+		if rule.HTTP != nil {
```

Comment:
How about inverting the logic here to avoid the extra nesting? I do note that the code is updating the map entry at the end which would be skipped if we do a `continue` here, does it really need to be updated even when there were no http entries in the rule? If so, then I guess it means a duplicated line before the `continue`, but I think the code would still be more readable without the extra nesting.